fdbcli/fdbcli.actor.cpp

```diff
@@ -669,6 +669,22 @@ std::string logBackupDR(const char *context, std::map<std::string, std::string>
 	return outputString;
 }
 
+int getNumOfZones(StatusObjectReader statusObjCluster) {
+	StatusObjectReader machinesMap;
+	std::set<std::string> zones;
+	if (statusObjCluster.get("machines", machinesMap)) {
+		int machineExclusions = 0;
```

Comment:
It doesn't look like `machineExclusions` is being used. Was there an intent to not count zones for excluded hosts? That seems like it may be a good idea.

Reply:
That was not my intent. It was a copy paste mistake.

Reply:
It may be helpful to remove excluded processes from consideration, since we won't be able to recruit on them anyway. I guess ideally, we wouldn't count any process which couldn't be recruited on regardless of the reason. So for example, if one zone contains only stateless processes, that zone doesn't help us with our diversity requirements.